docs/sftt/sftt-api-spec-new.md

```diff
@@ -539,6 +539,79 @@ The given `newUsername` is already in use.
 
 ### Get All Sites in GeoJSON
 
+## Import Router
+
+### Import Blocks
+
+`POST api/v1/protected/import/blocks`
+
+Used to import blocks into the database. Since blocks reference neigborhoods with a foreign key make sure all neighborhoods are imported first.
+
+#### Request Body
+
+```json
+{
+  "blocks": [
+    {
+      "block_id": INT,
+      "neighborhood_id": INT,
+      "lat": LONG,
+      "lng": LONG,
+      "coords": STRING,
+    },
+    ...
+  ]
+}
+```
+
+#### Responses
+
+##### `200 OK`
+
+Blocks imported succesfully.
+
+!!! missing "Missing error response, unsure what code"
```

Comment:
A 400 if the fields aren't valid and a 401 if the calling user isn't a super admin?